jobs/pacman-rule-engine-2.0/cloudresources/lambda/pac_submit_batch_job.py

```diff
@@ -46,18 +76,19 @@ def processJsonInput(event,context):
 def submit_to_batch(jobQueue,jobName,jobDefinition,containerOverrides,parameters):
 
     try:
-            # Submit a Batch Job
-            response = batch.submit_job(jobQueue=jobQueue, jobName=jobName, jobDefinition=jobDefinition,
-                                        containerOverrides=containerOverrides, parameters=parameters)
-            # Log response from AWS Batch
-            print("Response: " + json.dumps(response, indent=2))
-            # Return the jobId
-            jobId = response['jobId']
-            return {
-                'jobId': jobId
-            }
+        # Submit a Batch Job
```

Comment:
cq-colletor batch will look for different job-definition?  or it makes use of the collector job definition